ethcore/res/ethereum/classic.json

```diff
@@ -25,7 +25,7 @@
 		"accountStartNonce": "0x00",
 		"maximumExtraDataSize": "0x20",
 		"minGasLimit": "0x1388",
-		"networkID" : "0x1",
+		"networkID" : "0x3d",
```

Comment:
Currently there's no separate setting. This value is used for both network protocol id and EIP-155 transaction signing id